### ISSUE TITLE
Setting datePublished in ModifyQuery isn't working

### ISSUE THREAD

**Issue by eborisowIQVIA** [Bynder/bynder-java-sdk#65]
Trying to set the last published date for an uploaded item:

MediaModifyQuery modifyQuery = new MediaModifyQuery(<media id>);
					
TimeZone tz = TimeZone.getTimeZone("UTC");
DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
df.setTimeZone(tz);
String timeAsISO = df.format(UtilDateFormatter.getFormattedDate(new Date()));
						
System.out.println("Date of asset is: {}", timeAsISO);
modifyQuery.setDatePublished(timeAsISO);

Response<Void> modifyResponse = assetService.modifyMedia(modifyQuery).blockingSingle();

Basically, it fails with resulting date of: 2021-08-05T00:00Z

**Comment by githuib** [comment]
Hi @eborisowIQVIA,

When I try your example, the API responds with a 400 and with the message "Date format not ISO8601-compliant".
This is because the seconds are missing, adding those to the format string should work: `"yyyy-MM-dd'T'HH:mm:ss'Z'"`

FYI the snippet that worked for me:
```
Response<Void> modifyResponse = assetService.modifyMedia(
    new MediaModifyQuery("some valid asset ID").setDatePublished(
        new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").format(new Date())
    )
).blockingSingle();
```
